<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="btn_ok">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_picture">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>480</width>
      <height>270</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_url">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_download">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>541</width>
      <height>164</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbl_title">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_author">
       <property name="text">
        <string>Author</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_resolution">
       <property name="text">
        <string>Resolution</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_channelname">
       <property name="text">
        <string>Channel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_view">
       <property name="text">
        <string>Views</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_mime">
       <property name="text">
        <string>Video type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_filesize">
       <property name="text">
        <string>Filesize</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_length">
       <property name="text">
        <string>Length</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>40</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <item row="1" column="0">
      <widget class="QRadioButton" name="Rb_Audio">
       <property name="text">
        <string>Audio</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="Rb_Video">
       <property name="text">
        <string>Video</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>566</x>
      <y>0</y>
      <width>31</width>
      <height>571</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_download_all">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Download All Video </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>887</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
